src/irm.cpp

```diff
@@ -110,35 +104,22 @@ usage() {
         " ",
         "Remove one or more data-object or collection from iRODS space. By default, ",
         "the data-objects are moved to the trash collection (/myZone/trash) unless",
-        "either the -f option or the -n option is used.",
+        "the -f option is used.",
         " ",
         "Registered non-vault replicas are never deleted from the filesystem. They ",
         "are unregistered and left on disk as-is.",
         " ",
-        "The -U option allows the unregistering of the data object or collection",
-        "without deleting the physical file. Normally, a normal user cannot",
-        "unregister a data object if the physical file is located in a resource",
-        "vault. The acNoChkFilePathPerm rule allows this check to be bypassed.",
-        " ",
         "There is no -R option (remove replica from a named resource) at this time.",
         "Please use itrim (with the -S option) instead.",
         " ",
```

Comment:
This should be changed to say something like:
```bash
To remove a specific replica, use itrim.
```
We don't have to mention -R or anything. They can read the help text for itrim.

@trel Thoughts?

Reply:
Can do

Reply:
Done